[DQL3-11] - fix command, and strip whitespace from validator

awset opened the pull request:
https://salsadigital.atlassian.net/browse/DQL3-11

added negation query, remove whitespace before and after word, and remove empty line

awset committed:
[DQL3-11] - fix command, and strip whitespace from validator

tino097 reviewed (approved):
LGTM

tino097 commented on ckanext/data_qld/user_creation/validators.py:
```diff
@@ -18,6 +18,6 @@ def data_qld_displayed_name_validator(key, data, errors, context):
         excluded_names = config.get('ckanext.data_qld.excluded_display_name_words', '').split('\r\n')
         for name in excluded_names:
-            if name.lower() in data[key].lower():
+            if name.strip() and name.strip().lower() in data[key].lower():
```
i think only `name.strip().lower()` is enough

MarkCalvert commented on ckanext/data_qld/user_creation/commands.py:
```diff
@@ -20,5 +21,9 @@ def command(self):
 
-        model.Session.query(model.User).filter(model.User.created < date_threshold).update({'fullname': 'Displayed name required'})
+        model.Session.query(model.User) \
+            .filter(sa.not_(model.User.fullname.like('data.qld%'))) \
```
You could use the `sqlalchemy.func.lower()` to convert fullname to lowercase
```
from sqlalchemy import func
.filter(sa.not_(func.lower(model.User.fullname).like('data.qld%'))) \
```

MarkCalvert committed:
[DQL3-11]
- Updated `data_qld_displayed_name_validator` to do a None check for name
- Updated `update_fullname` to use `func.lower` for fullname filter
